### PR TITLE
Add croatian language

### PR DESCRIPTION
**Pull request by muzena**

### CODE REVIEW

File: resources/language/Croatian/strings.po [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Language: hr\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"X-Generator: Poedit 2.0.4\n"
+
+msgctxt "Addon Summary"
+msgid "Netflix"
+msgstr "Netflix"
```
Comment (asciidisco) [inline resources/language/Croatian/strings.po]: Can you remove that `msgstr` contents, if `msgid` & `msgstr` are the same, `msgstr` should be empty

File: resources/language/Croatian/strings.po [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+msgid "Enable Auto Login"
+msgstr "Omogući automatsku prijavu"
+
+msgctxt "#30055"
+msgid "Profile"
+msgstr "Profil"
+
+msgctxt "#30056"
+msgid "ID"
+msgstr "ID"
```
Comment (asciidisco) [inline resources/language/Croatian/strings.po]: Can you remove that `msgstr` contents, if `msgid` & `msgstr` are the same, `msgstr` should be empty

**Comment by muzena** [comment]
@asciidisco 
Is now ok?

**Comment by asciidisco** [comment]
Looks good now, thanks 👍

**Pull request merged**